sportsref/ncaaf/teams.py

```diff
@@ -75,4 +75,28 @@ def sos(self, year):
             print "ERROR: NO SRS FOUND FOR {} in {}".format(self.teamID, year)
             return np.nan
 
-# TODO - BEGIN HERE
+    def get_roster(self, year):
+        """ Returns the roster for a team for the year
+
+        :param year: the year of the roster
+        :return: a dataframe with the player_id, class, and position
+        """
+        url = sportsref.ncaaf.BASE_URL +\
+              '/schools/{}/{}-roster.html'.format(self.teamID, str(year))
+        doc = pq(sportsref.utils.getHTML(url))
+        table = doc('#all_roster')
+        df = sportsref.utils.parseTable(table)
+        return df
+
+def get_all_college_teams():
```

Comment:
This is fine here; I had a similar function in the nba.teams module, but then I realized the teams change each year as teams are added. If this is a concern for ncaaf, it may make sense to make a similar function as a method within the ncaaf.Season class (that's how I solved it for nba, and I believe nfl).